test/Serilog.Tests/Context/LogContextTests.cs

```diff
@@ -38,7 +38,7 @@ public LogContextTests()
         [Fact]
         public void PushedPropertiesAreAvailableToLoggers()
         {
-            LogEvent lastEvent = null;
+            LogEvent lastEvent = null!;
```

Comment:
Since the null value is observable, would:

```csharp
LogEvent? lastEvent = null;
```

be more appropriate here, and in the similar cases below?

Reply:
since u dont assert.notnull i assumed `null!;`. should i add a assert.notnull?

Reply:
`Assert.NotNull(logEvent)` would be a pleasant addition :+1: .. I think there's an implicit `Assert.NotNull()` in many of these cases - dotting through the reference to assert on `Properties` will fail if the event's null (not very elegantly, of course :-))

Reply:
done